source/_data/plugins.yml

```diff
@@ -3034,3 +3034,10 @@
     - javascript
     - js
     - css
+- name: hexo-filter-inline-svg
+  description: A Hexo plugin that inline your svg in the img tag into HTML file.
+  link: https://github.com/widergao/hexo-filter-inline-svg
+  tags:
+    - filter
+    - inline
+    - svg
```

Comment:
Could you leave a blank line at the end of file?

Reply:
OK, how about now.

Reply:
It looks like you added two lines 😂

Reply:
https://eslint.org/docs/rules/eol-last

Reply:
I'm sorry, is it right this time? I used the online editor in github but I don't know why the number of lines is not the same as the real one.